manifests/pip/bootstrap.pp

```diff
@@ -0,0 +1,52 @@
+#
+# @summary allow to bootstrap pip when python is managed from other module
+#
+# @param version should be pip or pip3
+# @param manage_python if python module will manage deps
+#
+# @example 
+#   class { 'python::pip::bootstrap':
+#     version => 'pip',
+#   }
+class python::pip::bootstrap (
+  Enum['pip', 'pip3'] $version            = 'pip',
+  Variant[Boolean, String] $manage_python = false,
+) inherits ::python::params {
+  if $manage_python {
+    include ::python
+  } else {
+    $target_src_pip_path = $::osfamily ? {
```

Comment:
instead of the legacy topscope variable, could you please use `$facts['os']['family']`?